erpnext/hr/doctype/expense_claim/expense_claim.py

```diff
@@ -360,3 +362,33 @@ def get_expense_claim(
 	)
 
 	return expense_claim
+
+# api to create expense claim
+@frappe.whitelist()
+def create_expense_claim(description, user_id, expense_date, expense_claim_type, amount):
```

Comment:
Please add docstrings and comments explaining what the function does in brief, and the args that is passed to it (if any) and the values that it returns (if any). It will help in enhancing our code quality. Format should be: 

"""
Description of function .... 
Args: 
    argument: data_type -> what the argument contains 
    argument: data_type -> what the argument contains 
Return: 
    data_type -> what values it returns. 
""" 

For example, this is what my get_data function in financial_statements.py looks like: 

	"""
	Returns data requested by financial statements
	Args: 
		company: string -> Name of the company
		root_type: string -> Type of account i.e. Asset, Liability, Equity, Income, Expense
		balance_must_be: string -> type of balance in the account i.e. debit or credit
		period_list: list -> time frames across which the data is needed
	Returns: 
		out: list -> list of dictionaries containing details of accounts
	"""

Reply:
Kindly note that writing "api to create expense claim" as a comment for a function named "def create_expense_claim" adds no value to the doctring/comments. Kindly elaborate on the information that can already be extrapolated from the function name.